server/test/com/sixsq/slipstream/ssclj/resources/device_lifecycle_test.clj

```diff
@@ -77,6 +77,9 @@
                                   :powerPlugged      true
                                   :agentType         "Fog agent"
                                   :actuatorInfo      "It has Ambulance, Firetruck, Sirene, Traffic light"
+                                  :sensorInfo        "Inclinometer"
```

Comment:
Regarding the `sensorInfo` , one thing I'd like to ask you @sstanovnik ...isn't it true that, in the mF2C we are basically considering the 4type of sensor devices (for example one type is Inclinometer). But, are we really considering the various kind of sensor like (temp, humidity etc.)??

Reply:
We might only be actively talking about 4 sensor devices, yes. But I don't see the relation to `sensorInfo`, what do you mean?

Reply:
Well to know the `sensorInfo` the initial plan was quite simple...as you know automatically know the sensor type information from the sensor is quite difficult...so after the internal discussion, we found a way for knowing the information. We decided that, this sensor type information should be taken as the user type input information during the registration process...So, I remember that, during the registration process four types of sensors are considering....so that's why I'm asking it...

Reply:
If that's the case, we just need to have "inclination" instead of "inclinometer" as the input value. The problem with using "inclinometer" et al is that names get weird : "temperature" is "thermometer", which is fine, but "distance" is "odometer" and "humidity" is "hygrometer", which are harder to understand at a glance, IMO. These also suggest that an "inclinometer" is the sole function of the sensor, whereas the device might also measure acceleration and velocity. 

My vote is for quantity names, not quantity measurement apparatus names.